cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -185,7 +185,7 @@ public void createCollection(String dbName, String collectionName, long collecti
                 .append("entityType", 1)
                 .append("attrName", 1)
                 .append("recvTime", 1);
-            options = new IndexOptions();
+            options = new IndexOptions().name("index");
```

Comment:
```suggestion
            options = new IndexOptions().name("cygnus_raw_optimization_index");
```

In addition, the index created a L197 should have a different name, e.g: "cygnus_raw_expiration_index"

Reply:
Fixed in 64d4802